core/modules/layout/layout.module

```diff
@@ -139,17 +146,19 @@ function layout_menu() {
     'type' => MENU_NORMAL_ITEM,
     'weight' => -10,
     'file' => 'layout.flexible.inc',
-  ) + $base;
+  );
+  $items['admin/structure/layouts/settings/flexible-template/%layout_flexible_tempstore/delete'] += $base;
 
   $items['admin/structure/layouts/settings/flexible-template/%layout_flexible_tempstore/row/%/delete'] = array(
     'title' => 'Delete row',
     'page callback' => 'layout_flexible_template_delete_row',
-    'page arguments' => array(5,7),
+    'page arguments' => [5, 7],
```

Comment:
Although this wouldn't cause any problems (Backdrop requires PHP 5.6+), we currently still use `array()` syntax everywhere.